src/dialogs/more-info/controls/more-info-fan.js

```diff
@@ -164,26 +170,32 @@ class MoreInfoFan extends LocalizeMixin(EventsMixin(PolymerElement)) {
     });
   }
 
-  onDirectionLeft() {
+  onDirectionReverse() {
     this.hass.callService("fan", "set_direction", {
       entity_id: this.stateObj.entity_id,
       direction: "reverse",
     });
   }
 
-  onDirectionRight() {
+  onDirectionForward() {
     this.hass.callService("fan", "set_direction", {
       entity_id: this.stateObj.entity_id,
       direction: "forward",
     });
   }
 
-  computeIsRotatingLeft(stateObj) {
-    return stateObj.attributes.direction === "reverse";
+  computeIsRotatingReverse(stateObj) {
+    return (
+      stateObj.attributes.direction === "reverse" ||
```

Comment:
it's only `direction` There is however a bug and it never ends up in the states to begin with. Fixing it now.

Reply:
https://github.com/home-assistant/home-assistant/pull/26298